src/Microsoft.ML.Transforms/TextCatalog.cs

```diff
@@ -169,5 +169,47 @@ public static class TextCatalog
             params WordTokenizeTransform.ColumnInfo[] columns)
           => new WordTokenizingEstimator(Contracts.CheckRef(catalog, nameof(catalog)).GetEnvironment(), columns);
 
+        /// <summary>
```

Comment:
>summary [](start = 13, length = 7)

please update to be a better user-facing description.

Reply:
fixed

---
In reply to: [233581290](https://github.com/dotnet/machinelearning/pull/1410#discussion_r233581290) [](ancestors = 233581290)